airflow/dags/rt_views/gtfs_rt_thread_errors.sql

```diff
@@ -0,0 +1,100 @@
+---
+operator: operators.SqlToWarehouseOperator
+dst_table_name: "views.gtfs_rt_thread_errors"
```

Comment:
cc @atvaccaro here - since he's been doing some investigation with the logs too, just wondering if he has thoughts on naming here

Reply:
Yeah I think you could leave out `thread` and probably specify that it's the archiver log errors; otherwise we may confuse it with GTFS RT _data_ errors.